_maps/configs/NEU_pill_black.json

```diff
@@ -8,5 +8,7 @@
 	"job_slots": {
 		"Prisoner": 3
 	},
-	"cost": 10
+	"cost": 10,
+	"limit": 3,
```

Comment:
Hell i'd say drop this down to 2, the black pill is impossible to progress on and is literally just a grief machine